runtime/parachains/src/paras_inherent.rs

```diff
@@ -371,9 +377,6 @@ pub mod pallet {
 				Vec::new()
 			});
 
-			let backed_candidates = limit_backed_candidates::<T>(backed_candidates);
```

Comment:
Why is `limit_backed_candidates` removed?

Reply:
We limit backed candidates in create_inherent, not in enter now.